prov/sm2/src/sm2_attr.c

```diff
@@ -32,129 +33,117 @@
 
 #include "sm2.h"
 
-#define SM2_TX_CAPS (OFI_TX_MSG_CAPS | FI_TAGGED )
```

Comment:
TODO: Remove FI_LOCAL_COMM and FI_REMOTE_COMM (temp in here for testing with OMPI...)

Reply:
You need to drop FI_REMOTE_COMM  before the PR can be merged

Reply:
done